tests/markdown/real-world-case.md

```diff
@@ -11,7 +11,7 @@ Prettier is an opinionated code formatter with support for:
 * [JSX](https://facebook.github.io/jsx/)
 * [Flow](https://flow.org/)
 * [TypeScript](https://www.typescriptlang.org/)
-* CSS, [LESS](http://lesscss.org/), and [SCSS](http://sass-lang.com)
```

Comment:
We don't need to edit this file and the snapshot. 😂 https://github.com/prettier/prettier/pull/3083#discussion_r146147066

Reply:
Updated